packages/storybook/stories/va-checkbox-group.stories.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import {
+  getWebComponentDocs,
+  componentStructure,
+  propStructure,
+} from './wc-helpers';
+
+const checkBoxGroupDocs = getWebComponentDocs('va-checkbox-group');
+const checkbox = getWebComponentDocs('va-checkbox');
+
+export default {
+  title: 'Components/va-checkbox-group',
+  subcomponents: componentStructure(checkbox),
+};
```

Comment:
Can you include the analytics event in here so it will show up on the "Actions" tab? 

https://github.com/department-of-veterans-affairs/component-library/blob/c8adb1152dd3db3d9fd68712da3bb445debb6fa5/packages/storybook/stories/va-loading-indicator.stories.jsx#L9-L13

With form controls we may also want to include the change events, but that can be something to address later.

Reply:
Added